embedded-graphics/README.md

```diff
@@ -15,41 +50,41 @@ If you need more deeper/more personalised help, please check out the [embedded-g
 
 ## [Documentation](https://docs.rs/embedded-graphics)
 
-A small 2D graphics library to draw things on embedded graphical LCDs, like the SSD1306 OLED display.
+![Embedded Graphics Simulator example screenshots](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/assets/simulator-demo.png)
 
-This crate aims to make drawing 2D graphics primitives super easy. It currently supports the
-following built in items:
 
-- Raw data images
-- Primitives
-  - Lines
-  - Rectangles (and squares)
-  - Circles
-  - Triangles
-- Text with multiple fonts
-
-Additional functionality provided by external crates:
+## Additional functions provided by external crates
```

Comment:
Can we move this section closer/underneath the built in features list? I think it's useful to make it easier to discover extensions to e-g.

Reply:
I've now tried to incorporate the additional features provided by the external creates in the main feature overview. Do you think that would work?

If we move the entire section the fancy screenshot image would move further down and maybe not as many people would notice it. One solution might be to use multiple smaller screenshots throughout the readme instead of one huge screenshot collage, that fills the entire screen.

Reply:
How about moving the two photos into their own wide and short image that can go under the `Embedded Graphics` heading? Or even centered underneath the badges at the top of the page.

The simulator screenshots could be moved/replaced with individual outputs on each example near the bottom of the page.

Reply:
> Do you think that would work?

Hm, not really. I was thinking of two separate bullet lists, one after the other. It should take up nearly the same amount of vertical space as your change just now and we can go back to having a nice list of links for external crates.

Reply:
> How about moving the two photos into their own wide and short image that can go under the `Embedded Graphics` heading? Or even centered underneath the badges at the top of the page.

That sounds like a good idea. Can you prepare that image, because I don't have access to the full size versions of the images?

Reply:
> Hm, not really. I was thinking of two separate bullet lists, one after the other. It should take up nearly the same amount of vertical space as your change just now...

I get your point, but having a link list inside the overview of the crate isn't ideal either. The 4 links aren't to disturbing, but if the lists gets longer this isn't the ideal place for it IMO.

> ... and we can go back to having a nice list of links for external crates.

We still have that, only a bit further down the page. But if we remove the large screenshot we could move the "Additional functions provided by external crates" sections directly after the introduction.

Reply:
> Can you prepare that image

I've pushed a banner here: https://github.com/jamwaffles/embedded-graphics/blob/readme-images/assets/banner-photo.png it's a 2x scaling of the 100% width of a Github readme so should hopefully fit nicely. I added drop shadows because I'm not sure what else to do to make it look good tbh. Looks alright though.

If you pull it in, please take the GIMP XCF too: https://github.com/jamwaffles/embedded-graphics/blob/readme-images/assets/banner-photo.xcf it's massive, but that gives us scope to reuse the images later.

Reply:
As for the other layout stuff, I'll leave it to your better judgement and see what it looks like with the images split/moved around a bit.

Reply:
> If you pull it in, please take the GIMP XCF too: https://github.com/jamwaffles/embedded-graphics/blob/readme-images/assets/banner-photo.xcf it's massive, but that gives us scope to reuse the images later.

I don't like the idea of doubling the size of the repo just for one image, but it would be useful to have these available. Can we add a separate repo just for assets?

Reply:
A separate repo seems a bit overkill. I could just keep it in the branch I linked to, or delete it? It's not difficult to recreate that banner tbh. How about I export a high-ish resolution PNG and replace the XCF with it? That gives a decent tradeoff between size and future usability, no?

Reply:
I did tweak your image a bit, because I didn't like the look of images with different heights in one row. And I also removed the shadow to gain some additional with and have the left and right edges of the image be aligned with the rest of the content. It might look a little more plain now, but I think it's OK. Or do you prefer to change it back?

> A separate repo seems a bit overkill.

Sure, having a repo just for this one image is overkill. My idea was to have a place to store nice e-g related images that can be reused for different purposes like docs or if someone wants to write a blog post about e-g. But that's probably also overkill if we only have a few images.

Reply:
> but I think it's OK

Me too haha, thanks for fixing my botch job.